public/scss/base.scss

```diff
@@ -8,6 +8,10 @@ body {
   padding: 1rem 2rem;
 }
 
+.admin-title {
+  margin: 1rem 2rem;
+}
```

Comment:
there's already style defined for .admin-changelist__header. We should probably refactor it to make it reusable and use it on the form pages